FilePlanDescriptors for RetentionLabels (#20879)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/security-labelsroot.md

---
title: "labelsRoot resource type"

> description: "Represents the entry point for retention labels resource type."

Reviewer: ```suggestion
description: "Represents the entry point for the retentionLabels resource type."
```

author: "sseth"
ms.localizationpriority: medium
ms.prod: "security"
doc_type: resourcePageType
---

# labelsRoot resource type

Namespace: microsoft.graph.security

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Represents the entry point for retention labels resource type.

Reviewer: ```suggestion
Represents the entry point for the retentionLabels resource type.
```

Inherits from [microsoft.graph.entity](../resources/entity.md).

## Methods
|Method|Return type|Description|
|:---|:---|:---|

> |[List retentionLabels](../api/security-retentionlabel-list.md)|[microsoft.graph.security.retentionLabel](../resources/security-retentionlabel.md) collection|Get a list of the [retentionLabel](../resources/security-retentionlabel.md) objects and their properties.|

Reviewer: ```suggestion
|[List retentionLabels](../api/security-retentionlabel-list.md)|[microsoft.graph.security.retentionLabel](../resources/security-labelsroot-list-retentionlabel.md) collection|Get a list of the [retentionLabel](../resources/security-retentionlabel.md) objects and their properties.|
```

> |[List retentionLabels](../api/security-retentionlabel-list.md)|[microsoft.graph.security.retentionLabel](../resources/security-labelsroot-list-retentionlabel.md) collection|Get a list of the [retentionLabel](../resources/security-retentionlabel.md) objects and their properties.|

Reviewer: ```suggestion
|[List retentionLabels](../api/security-labelsroot-list-retentionlabel.md)|[microsoft.graph.security.retentionLabel](../resources/security-labelsroot-list-retentionlabel.md) collection|Get a list of the [retentionLabel](../resources/security-retentionlabel.md) objects and their properties.|
```

> |[Create retentionLabel](../api/security-retentionlabel-post.md)|[microsoft.graph.security.retentionLabel](../resources/security-retentionlabel.md)|Create a new [retentionLabel](../resources/security-retentionlabel.md) object.|

Reviewer: ```suggestion
|[Create retentionLabel](../api/security-labelsroot-post-retentionlabel.md)|[microsoft.graph.security.retentionLabel](../resources/security-retentionlabel.md)|Create a new [retentionLabel](../resources/security-retentionlabel.md) object.|
```

|[List departments](../api/security-departmenttemplate-list.md)|[microsoft.graph.security.departmentTemplate](../resources/security-departmenttemplate.md) collection|Get the departmentTemplate resources from the departments navigation property.|
|[Create departmentTemplate](../api/security-departmenttemplate-post.md)|[microsoft.graph.security.departmentTemplate](../resources/security-departmenttemplate.md)|Create a new departmentTemplate object.|
|[List filePlanReferences](../api/security-fileplanreferencetemplate-list.md)|[microsoft.graph.security.filePlanReferenceTemplate](../resources/security-fileplanreferencetemplate.md) collection|Get the filePlanReferenceTemplate resources from the filePlanReferences navigation property.|
|[Create filePlanReferenceTemplate](../api/security-fileplanreferencetemplate-post.md)|[microsoft.graph.security.filePlanReferenceTemplate](../resources/security-fileplanreferencetemplate.md)|Create a new filePlanReferenceTemplate object.|

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |

Reviewer: Does the resource have properties, or should it be "none"? Please update the section accordingly.
